tests/test_secondary.py

```diff
@@ -0,0 +1,721 @@
+"""
+<Program Name>
+  test_secondary.py
+
+<Purpose>
+  Unit testing for uptane/clients/secondary.py
+  Much of this is copied from test_primary.py and then modified slightly
+"""
+from __future__ import print_function
```

Comment:
Debugging reasons, but you're right: I shouldn't spam. I'll strip it out here and try to remember to do likewise in the other test modules.